packages/ts/react-i18n/test/i18n.spec.tsx

```diff
@@ -176,6 +182,21 @@ describe('@vaadin/hilla-react-i18n', () => {
         await i18n.setLanguage('de-DE');
         expect(effectSpy.calledOnceWith('de-DE', 'Stadt')).to.be.true;
       });
+
+      it('should run effects when initialized changes', async () => {
```

Comment:
Should we also check for not running effects when reconfigured after initializing?

Reply:
We haven't really defined what should happen if developers call `configure` multiple times, or if that should even be allowed. Besides, re-running effects doesn't seem problematic on its own?